### ISSUE TITLE
docs are out of date

### ISSUE THREAD

**Issue by joejulian** [samsung-cnct/k2cli#88]
```
$ pwd
/home/jjulian/work/cluster
$ k2cli generate -k $PWD/staging.yaml
Pulling image 'quay.io/samsung_cnct/k2:latest' ███▒▒▒▒▒▒▒ Complete
Generated aws config at /home/jjulian/.kraken/config.yaml
```
The expectation was that the configuration would be written to $PWD/staging.yaml as per the cli switch option.

**Comment by coffeepac** [comment]
this is a bug but not the one you think it is.  '-k' takes a parameter, which means the parameter you pass in will get consumed there and not be available for use by the generate subcommand.  the value being passed into '-k' should be used for both.  I think there's a bug for inconsistent references to the config file.  but, example:


> MacBook-Pro:hoof pat$ k2cli generate -k /Users/pat/scratch/hoof/file /Users/pat/scratch/hoof/file2
Pulling image 'quay.io/samsung_cnct/k2:latest' ███▒▒▒▒▒▒

**Comment by joejulian** [comment]
Which means that the self-generated documentation is both incorrect and not backward compatible.

I would prefer that both the documentation is corrected and, as @coffeepac beat me to saying, the switch be used consistently.

```
$ k2cli --help
k2cli is a command line interface for K2
	kubernetes cluster provisioner. K2 documentation is available at:
	https://github.com/samsung-cnct/k2

Usage:
  k2cli [command]

Available Commands:
  cluster     K2 cluster actions
  docs        Generate markdown docs
  generate    Generate a K2 config file
  tool        Use tools with K2 cluster
  version     Display cli version

Flags:
  -d, --docker-host string   docker host address (default "unix:///var/run/docker.sock")
  -i, --image string         k2 container image (default "quay.io/samsung_cnct/k2:latest")
  -k, --k2config string      config file for k2cli (default "/home/jjulian/.k2cli.yaml")
  -a, --keep-alive           keep stopped containers.
  -w, --log-path string      Save output output of container action to path
  -x, --log-success          Display full action logs on success
  -o, --output string        K2 output folder (default "/home/jjulian/.kraken")
  -t, --timeout int          timeout (in seconds) for container actions (default 1200)
  -v, --verbosity            Verbose output

Use "k2cli [command] --help" for more information about a command.
```

**Comment by davidewatson** [comment]
The -k option is intended to read a k2cli configuration file which is not currently implemented: `  -k, --k2config string      config file for k2cli (default "/Users/dwat/.k2cli.yaml")` It should be removed, or fixed.

**Comment by DStorck** [comment]
can we re-generate the docs? they are out of date and there have been several flag changes.

**Comment by davidewatson** [comment]
@DStorck: Command line arguments are documented by golang code, so a PR is required if there are inaccuracies.

**Comment by DStorck** [comment]
My above comment was unclear - I was referring to the docs on our k2cli github page , which are auto-generated by cobra and haven't been updated in 8 months.

**Comment by coffeepac** [comment]
will be resolved by #91
